scripts/zones/The_Shrouded_Maw/mobs/Diabolos.lua

```diff
@@ -49,6 +49,8 @@ entity.onMobSpawn = function(mob)
     mob:setLocalVar("TileTriggerHPP", triggerVal) -- Starting point for tile drops
     mob:setLocalVar("Area", area)
     mob:setMobMod(xi.mobMod.DRAW_IN, 1)
+    mob:setMobMod(xi.mobMod.DRAW_IN_INCLUDE_PARTY, 1)
+    mob:setMod(xi.mod.DMGPHYS, -5000)
```

Comment:
Thats a huge damage reduction - this seems ok for balancing wise?

Reply:
> Thats a huge damage reduction - this seems ok for balancing wise?

From my understanding avatar mobs (including diabolos) should have this 50% phys damage reduction (as per the linked issue). I looked to see if it was being applied somewhere else to all avatar mobs but could not find it so added it to diabolos.

Reply:
Sounds good to me, lets just make a note to really test this fight again with a full party before it goes to a live server.

Reply:
FYI at minimum, the fight is way too easy right now so a 50% is probably not as bad as it sounds.